app/models/user.rb

```diff
@@ -3,6 +3,8 @@ class User < ApplicationRecord
   has_many :likes, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
 
+  validates :name, presence: true
+
```

Comment:
- [ ] According to the project requirements you should add the following validations for the User model: PostsCounter must be an integer greater than or equal to zero. Kindly add this it should be something like this:
`validates_numericality_of :posts_counter, only_integer: true, greater_than_or_equal: 0`

Reply:
@BenMukebo , This has now been implemented